cmd/ak/common/svc.go

```diff
@@ -0,0 +1,23 @@
+package common
+
+import (
+	"go.autokitteh.dev/autokitteh/backend/svc"
+	"go.autokitteh.dev/autokitteh/internal/backend/configset"
+)
+
+var mode configset.Mode
+
+func SetMode(m string) (err error) {
```

Comment:
An exported function that merely controls a private var? Isn't this a Go smell/antipattern?